rocky/reports/templates/partials/report_severity_totals.html

```diff
@@ -0,0 +1,18 @@
+{% load i18n %}
+
+<section>
+    <div>
+        <h3>{% translate "Summary" %}</h3>
+        <div class="column-2">
+            {% include "partials/report_severity_totals_table.html" %}
+
+        </div>
+        {% if data.total_occurrences > 0 %}
+            <div class="nota-bene">
+                <span>{% translate "See" %}: <a href="https://nvd.nist.gov/vuln-metrics/cvss"
```

Comment:
I think that including `:` in the translated string makes it easier for translator to know what the context is.

```suggestion
                <span>{% translate "See:" %} <a href="https://nvd.nist.gov/vuln-metrics/cvss"
```

(The translation files are need to be updated when the suggestion is committed)